src/node/checkWin.js

```diff
@@ -0,0 +1,96 @@
+const constants = require('./constants.js');
+const Point = require('./point.js');
+
+function checkWin(userMove, board) {
+  if (
+    checkIfMoveWinsVertically(userMove, board) ||
+    checkIfMoveWinsHorizontally(userMove, board) ||
+    checkIfMoveWinsDiagonallyNEtoSWaxis(userMove, board) ||
+    checkIfMoveWinsDiagonallyNWtoSEaxis(userMove, board)
+  ) {
+    return true;
+  }
+  return false;
+}
+
+function checkIfMoveWinsVertically(userMove, board) {
+  const playerSymbol = board.getSymbol(userMove);
+  for (
+    let pointIterator = Point(userMove.row, 0);
+    pointIterator.col < constants.boardSize;
+    pointIterator.col += 1
+  ) {
+    if (board.getSymbol(pointIterator) !== playerSymbol) {
+      return false;
+    }
+  }
+  return true;
+}
+
+function checkIfMoveWinsHorizontally(userMove, board) {
+  const playerSymbol = board.getSymbol(userMove);
+  for (
+    let pointIterator = Point(0, userMove.col);
+    pointIterator.row < constants.boardSize;
+    pointIterator.row += 1
+  ) {
+    if (board.getSymbol(pointIterator) !== playerSymbol) {
+      return false;
+    }
+  }
+  return true;
+}
+
+function checkIfMoveWinsDiagonallyNWtoSEaxis(userMove, board) {
```

Comment:
Hmm whoever use this function will never care that you implemented this check by iterating from NW ([0][0]) to SE([size][size]). Calling it `checkMainDiagonal` and `checkSecondaryDiagonal` should be clear enough, but thas my opinion though.